packages/server/graphql/mutations/helpers/isStartMeetingLocked.ts

```diff
@@ -1,7 +1,16 @@
+import {Threshold} from '~/types/constEnums'
 import {DataLoaderWorker} from '../../graphql'
 
 const isStartMeetingLocked = async (teamId: string, dataLoader: DataLoaderWorker) => {
   const team = await dataLoader.get('teams').loadNonNull(teamId)
+  const organization = await dataLoader.get('organizations').load(team.orgId)
+
+  const {lockedAt, name: organizationName} = organization
+
+  if (lockedAt) {
```

Comment:
+1 I think this should come after the `lockMessageHTML`. That way, someone from sales can add a custom lock message even if `lockedAt` is true